index.html

```diff
@@ -42,46 +34,184 @@ <h2 class="title-about">I'm a Software Developer</h2>
         
 
         <ul class="social-icon">
             <li>
                 <a href="https://github.com/ulises2607" class="soc-icon">
-                    <img src="Images/Social/GitHub.svg" alt="github-profile">
+                    <img src="Images/Main/Social/GitHub.svg" alt="github-profile">
                 </a>
             </li>
             <li>
                 <a href="https://www.linkedin.com/in/cesar-ulises-cappa-subelza/" class="soc-icon">
-                    <img src="Images/Social/Linkedin.svg" alt="linkedin-profile">
+                    <img src="Images/Main/Social/Linkedin.svg" alt="linkedin-profile">
                 </a>
             </li>
             <li>
                 <a href="https://www.angellist.com/" class="soc-icon">
-                    <img src="Images/Social/Angelist.svg" alt="Angelist-mainpage">
+                    <img src="Images/Main/Social/Angelist.svg" alt="Angelist-mainpage">
                 </a>
             </li>
             <li>
                 <a href="www.twitter.com" class="soc-icon">
-                    <img src="Images/Social/Twitter.svg" alt="twitter-mainpage">
+                    <img src="Images/Main/Social/Twitter.svg" alt="twitter-mainpage">
                 </a>
             </li>
             <li>
                 <a href="https://medium.com/" class="soc-icon">
-                    <img src="Images/Social/Medium.svg" alt="medium-mainpage">
+                    <img src="Images/Main/Social/Medium.svg" alt="medium-mainpage">
                 </a>
             </li>
 
         </ul>
-            
 
-        
+    </section>
+
+    <section class="recent-works">
+
+        <h1>My Recent Works</h1>
+
+        <hr>
+
+        <img src="Images/Work/Work-img-1.png" alt="Multi-post-stories">
+
+        <h2>Multi-Post Stories</h2>
+
+        <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required. 
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took 
+            a standard dummy text.
+        </p>
+
+        <ul class="tech">
+            <li><a href="#">CSS</a></li>
+            <li><a href="#">HTML</a></li>
+            <li><a href="#">Bootstrap</a></li>
+            <li><a href="#">Ruby</a></li>
+        </ul>
+
+        <a href="#" class="btn-works-1">See Projects</a>
+    </section>
+
+<!-- Project Cards -->
+    <section>
+        <div class="card-container">
+            <div class="card">
+                <h1 class="card-title">Profesional Art Printing Data</h1>
+                <p class="card-description">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <ul class="card-lang">
+                    <li>HTML</li>
+                    <li>CSS</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#" class="btn-works">See Project</a>
```

Comment:
👍👏 Great work on the button! It has a good overall style. However, there are a few improvements to align it with the design:

- [ ] Please make sure to add the hover and active styles to the button as per the design. This will enhance the user experience and provide visual feedback when interacting with the button. :pray:

![image](https://github.com/ulises2607/Portfolio/assets/55840999/9cf8eb59-218b-4383-8167-a3984178d8e4)

By incorporating these changes, you will ensure that the button meets the design requirements and delivers a consistent user experience. Keep up the fantastic work! 👌😊